django_filters/filters.py

```diff
@@ -656,7 +656,7 @@ def normalize_fields(cls, fields):
 
     def build_choices(self, fields, labels):
         ascending = [
-            (param, labels.get(field, pretty_name(param)))
+            (param, labels.get(field, _(pretty_name(param))))
             for field, param in fields.items()
         ]
         descending = [
```

Comment:
That's not a bad point. It's also a fairly trivial thing to add. Currently updating the PR to include this & your corresponding test.